remoteit/desktop
# DESK-1148 Add ip address connection param to account for lan sharing with certs

### Description


Add ip address connection param to account for LAN sharing with certs
Also added LAN Address connection attribute

### Test plan



1. Sign in
2. Go to connection
3. Click lan sharing
4. Turn it on
5. Create connection
5. Check you can connect from within the network with the LAN Address

### Ticket



<https://remoteit.atlassian.net/browse/DESK-1148>
